### PR TITLE
Ticket 21: fixed bug on adding tag inside posts.json file

### PR DESCRIPTION
**Pull request by seintun**
Please review on fixing bugs: tags key:value is added to posts.json

### CODE REVIEW

File: storage/posts.json [hunk 1]
```diff
@@ -71,4 +74,3 @@
             }
         ]
     }
-]
```
Comment (angelo-james) [inline storage/posts.json:74]: add a closing square bracket

**Review by angelo-james — CHANGES_REQUESTED** [review]
add a closing square bracket

File: storage/posts.json (hunk 1 above)
Comment (seintun) [reply storage/posts.json:74]: Seriously? I am sorry.

**Review by angelo-james — APPROVED** [review]
squirrel - good to merge

**Pull request merged**